### PR TITLE
Steam as of 14th Dec Fixes

### PR DESCRIPTION
**Pull request by sunhwapark**
Updates to:
ETC
ITEM
QUEST
QUEST_LV_0100
QUEST_LV_0400
SKILL

edit note: * There is potential conflicts in the Item.TSV

### CODE REVIEW

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -15985,7 +15985,7 @@ ITEM_20171128_015984	Primus Migatis Rod	프리머스 미간티스 로드
 ITEM_20171128_015985	Primus Pevordimas Rod	프리머스 페버디마스 로드				
 ITEM_20171128_015986	Primus Raffye Rod	프리머스 라파이 로드				
 ITEM_20171128_015987	Masinios Rod	마시니오스 로드				
-ITEM_20171128_015988	 Increases attack of the Consciousness property by 10%. {img green_up_arrow 16 16}5 {nl}Stacks up to 10 times.	 - 스킬 공격시 10% 확률로 정신 {img green_up_arrow 16 16}5 {nl}최대 10중첩				
+ITEM_20171128_015988	 Attacks using skills have a 10% chance of increasing SPR {img green_up_arrow 16 16}5 SPR{nl}Stacks up to 10 times.	 - 스킬 공격시 10% 확률로 정신 {img green_up_arrow 16 16}5 {nl}최대 10중첩
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: It mentions SPR twice. Only once is necessary,
You put a dot in the second sentence, but not in ITEM_20171128_016016. Consistency. Dot or no dot?

File: EnglishTranslation-master/ITEM.tsv [hunk 2]
```diff
@@ -16916,15 +16916,15 @@ ITEM_20171128_016915	A refined unidentified crystal core that mutates both organ
...
 ITEM_20171128_016920	Old Story of Laima. right click to read 	라이마에 관한 오래된 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
-ITEM_20171128_016921	Story of Laima and Jesti, right click to read 	라이마와 제스티에 관한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
+ITEM_20171128_016921	Story of Laima and Gesti, right click to read 	라이마와 제스티에 관한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
 ITEM_20171128_016922	Story of Premier Emminant, right click to read 	프리미어 에미넌트에 관한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
 ITEM_20171128_016923	Manuel for an administrator of Fantasy Library. right click to read 	환상도서관 관리자를 위한 지첨서. 마우스 우클릭으로 읽을 수 있습니다. 				
 ITEM_20171128_016924	Story of Martis, right click to read 	미르티스에 대한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
 ITEM_20171128_016925	Story of Jetsi, right click to read 	제스티에 대한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
 ITEM_20171128_016926	Story of Nebulas, right click to read 	네뷸라스에 대한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
-ITEM_20171128_016927	Story of Solcom, right click to read 	솔콤에 대한 이야기. 마우스 우클릭으로 읽을 수 있습니다. 				
+ITEM_20171128_016927	Story of Solcomm, right click to read 	솔콤에 대한 이야기. 마우스 우클릭으로 읽을 수 있습니다.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: right click -> right-click (Can you do a mass replace, please?)

File: EnglishTranslation-master/ITEM.tsv [hunk 3]
```diff
@@ -17000,180 +17000,180 @@ ITEM_20171128_016999	Increases EXP up to character level 180. Can be used at lev
...
+ITEM_20171128_017115	It is said to be a mirror used in ceremonies and rituals as the sacred material of the Goddess Jurate, sealed by the Demons. Its reflection is clouded by an evil aura.	마족에 의해 봉인된 유라테 여신의 성물로 제사나 의식에 사용되던 거울이라고 한다. 사악한 기운에 의해 탁해져 있다.				
 ITEM_20171128_017116	Jurate's first Holy Artefact	유라테의 첫 번째 성물				
 ITEM_20171128_017117	The sacred treasure of Jurate that tradition person Ugnius had. It was said that it was a necklace worn by a priest who conducted rituals and ordinances.	전승자 우그뉴스가 가지고 있던 유라테의 성물. 제사나 의식을 주관하는 사제가 착용하는 목걸이였다고 한다.				
-ITEM_20171128_017118	Jurate's second Holy Artefact	유라테의 두 번째 성물				
-ITEM_20171128_017119	The Second Earthen of Jurate. Although the Asmodians sealed it, it is a mirror that was used in rituals and rituals as a sanctuary that washes away the demonic energy with the aura of the sea.	유라테의 두 번째 성물. 마족이 봉인해두었으나, 바다의 기운으로 마기를 씻어낸 성물로 제사나 의식에서 사용되던 거울이라고 한다.				
+ITEM_20171128_017118	Jurate's Second Holy Artefact	유라테의 두 번째 성물
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Artefact is British. Since we use 'untradable', I would go for Artifact.

also: Jurate's first Holy Artefact

**Comment by sunhwapark** [comment]
@Sunaries 
Thanks for the check, changes applied.
Some further changes have been added, majority being strings i could pull from the live client.
Should be :+1: if there are no outstanding changes required.

**Comment by sunhwapark** [comment]
Closing to move to the 20th Update, it's kind of too big !

**Pull request closed without merging**